schema/codelists/partyRole.csv

```diff
@@ -10,3 +10,9 @@ payee,Payee,A party in receipt of a payment from a transaction.,
 reviewBody,Review body,A party responsible for the review of this procurement process. This party often has a role in any challenges made to the contract award.,
 interestedParty,Interested party,"A party that has expressed an interest in the contracting process: for example, by purchasing tender documents or submitting clarification questions.",
 contractImplementationManager,Contract implementation manager,The organization responsible for managing the implementation of the contract on behalf of the buyer. (This may be different from the procuring entity that manages the contracting process.),
+mediationBody,Mediation body,The body responsible for mediation procedures.,
+centralPurchasingBody,Central purchasing body,"The procuring entity providing centralized purchasing activities and, possibly, ancillary purchasing activities.",
```

Comment:
@JachymHercher This was added to the EU extension for I.2 "The contract is awarded by a central purchasing body" from the current EU forms. With respect to #1180, does it make sense to use this code? Does it match any of the 4 codes proposed in #1180?

Reply:
'centralPurchasingBody' covers both 'wholesaler' and 'contract intermediary' from https://github.com/open-contracting/standard/issues/1180#issue-792561618. In eForms, the codes only exist in their split form (see [cpb-acq, cpb-awa](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/organisation-role)), but in the TED 2.0.9 forms they only exist as one code. To be able to map to both, we would either need to include all three codes, or decide that for TED 2.0.9, a CPB should be marked by including both the 'wholesaler' and 'contract intermediary'. The latter option avoids clutter in OCDS, but might be less obvious to users (not sure how many there are for TED 2.0.9 though).

Reply:
What was the use case for splitting CPB into two codes?

Reply:
Noting that we can also leave/restore 'centralPurchasingBody' in the EU extension until we resolve #1180.

Reply:
I have removed 'centralPurchasingBody' from the PR.

Reply:
> What was the use case for splitting CPB into two codes? What need does it satisfy? Is it just to be maximally accurate from the publisher's perspective, or does it have analytical value?

Primarily policymaking: wholesale and intermediary are two very different ways of doing/organizing central purchasing and being able to distinguish them is the first step to knowing how they are used and, ultimately, which one is better in which contexts (e.g. for which types of purchases). Speculatively, wholesale/intermediary might also turn out to be a "shorthand" way of communicating certain information to the market such as single/multiple place of delivery, single/multiple payments (and the related delays in payments), etc. These can always be specified in more detail elsewhere of course.